courses/basic-algorithm-scripting/scripts.js

```diff
@@ -0,0 +1,195 @@
+// Convert Celsius to Fahrenheit
+function convertToF(celsius) {
+    let fahrenheit = celsius*(9/5) + 32;
+    return fahrenheit;
+}
+
+convertToF(30);
+
+// Reverse a String
+function reverseString(str) {
+    str = str.split('').reverse().join('');
+    return str;
+}
+
+reverseString('hello');
+
+// Factorialize a Number
+function factorialize(num) {
+
+    if(num === 0){
+        num = 1;
+    } else{
+        for (let i = num - 1; i > 0; i--){
+            num *= i;
+        }
+    }
+    return num;
+}
+
+factorialize(5);
+
+// Find the Longest Word in a String
+function findLongestWordLength(str) {
+
+    let temp = '';
+
+    str.split(' ').forEach(function(word){
+        if(temp.length < word.length){
+            temp = word;
+        }
+    });
+
+    return temp.length;
+}
+
+findLongestWordLength('The quick brown fox jumped over the lazy dog');
+
+// Return Largest Numbers in Arrays
+function largestOfFour(arr) {
+    let temp = [];
+
+    for(let i = 0; i< arr.length; i++){
```

Comment:
Could this part be done also with a `forEach`, instead of `for`?

Reply:
Using [Reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) will save you a lot of code here.